src/Http/Controllers/Webhooks/CustomerDeleteController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace StatamicRadPack\Shopify\Http\Controllers\Webhooks;
+
+use Illuminate\Http\Request;
+use Statamic\Facades\User;
+use StatamicRadPack\Shopify\Events;
+
+class CustomerDeleteController extends WebhooksController
+{
+    public function __invoke(Request $request)
+    {
+        $hmac_header = $request->header('X-Shopify-Hmac-Sha256');
+        $data = $request->getContent();
+        $verified = $this->verify($data, $hmac_header);
+
```

Comment:
I usually put webhook validation in a middleware...

Reply:
Yeah it hasnt been changed since we inherited it. Probably better in a separate PR.